News/DataAccess/Repo/PasswordRepo.cs

```diff
@@ -11,4 +11,15 @@ public PasswordRepo(DbContext context)
         : base(context)
     {
     }
+
+    public async Task<PasswordEntity> GetByIdAsync(
+        Guid? id,
+        CancellationToken cancellationToken
+        )
+    {
+        var entity = await _dbSet
+            .FirstOrDefaultAsync(e => e.Id == id, cancellationToken);
+
+        return entity;
+    }
```

Comment:
Are you sure about it? We can just change generic and dont need to do the same logic here.

Reply:
Yes, because in the generic class, Guid is required for all GetByIdAsync methods and cannot be null. Only here, Guid? is temporarily used as null, so it’s better to implement the specialized logic only in this class.

Reply:
Ok I guess @AvaNice can deside how to implement it in this case.